data-reconciliation-app/Makefile

```diff
@@ -80,7 +80,7 @@ clean: ## 🧹 Clean the working folders created during build/demo
 	@rm -rf dist
 
 # Generate a MS-Idp AAD (Azure active directory) JWT (app-to-app) token for the application testing;
-generate-jwt-token: # Generate an Azure AD access tokens for authentication
+generate-jwt-token-ms: # Generate an Azure AD access tokens for authentication
```

Comment:
I've moved this in my PR! Sorry!

Reply:
I've merged your changes to my branch and fixed this one